src/components/layout/mdx-layout.module.css

```
@@ -0,0 +1,49 @@
+  font-weight: 400;
+}
+
+.handwritingArticle {
+  font-family:
+    var(--font-content-handwriting), var(--font-sans), system-ui, sans-serif;
+  font-size: 1.0625rem;
+  line-height: 1.9;
+  word-break: keep-all;
+  overflow-wrap: break-word;
+  font-synthesis-weight: none;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.14.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)